### ISSUE TITLE
Change language in-app

### ISSUE THREAD

**Issue by TrianguloY** [TrianguloY/UrlChecker#45]
Add an option to allow the user to change the language of the app by choosing from a list inside the app itself, without having to modify the system settings.

Will probably require a new app settings sub screen.

**Comment by Ilithy** [comment]
> Will probably require a new app settings sub screen

_If you add this (General settings), it would be nice to have also in these settings the possibility to change the theme of URLCheck (light/dark/system)._

**Comment by TrianguloY** [comment]
Yup. There are several app-related settings planned for the future, and thats exactly why I made a new settings screen (currently on a temporal branch until I have something to fill it with!).

**Comment by TrianguloY** [comment]
Ah wait, I though there were already an issue for the theme selector. Feel free to create one (I'll do it later otherwise)

**Comment by Ilithy** [comment]
Sorry for posting this here, this is neither good nor practical, I didn't dare to create an issue for this, it's done (#94);
I also think there was already one subject related to themes, but I did not find it either (maybe also in an issue on the peripheral subject)
